config/routes.rb

```diff
@@ -635,10 +635,12 @@
 
       resources :notes, only: %i[index show new create], as: :application_choice_notes
 
-      resources :interviews, only: %i[new index], as: :application_choice_interviews do
+      resources :interviews, only: %i[new edit index], as: :application_choice_interviews do
         collection do
           get '/new/check', to: 'interviews#check'
           get '/confirm', to: 'interviews#commit'
+          get '/check/:id', to: 'interviews#check', as: :check
+          get '/update/:id', to: 'interviews#update', as: :update
```

Comment:
```suggestion
           put :update
```

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3993/commits/00bd67a555c31f91758853fd2daf472b767f1923